anthos-bm-gcp-terraform/README.md

```diff
@@ -0,0 +1,155 @@
+
+## Anthos Baremetal on GCE with Terraform
+
+This repo is work-in-progress to replicate the [official AnthosBM setup docs](https://cloud.google.com/anthos/clusters/docs/bare-metal/1.6/try/gce-vms#vms) using Terraform. The intension is to automate many of the copy-paste steps available there with sufficient details and provide a friction free first experience.
+
+### Pre-requisites
+
+- A workstation with access to internet _(i.e. Google Cloud APIs)_ with the following installed
+  - [git](https://www.atlassian.com/git/tutorials/install-git)
+  - [gcloud](https://cloud.google.com/sdk/docs/install)
```

Comment:
I'm very interested in figuring out if https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project can replace gcloud in this README.

Reply:
Currently, the only place we use `gcloud` is to SSH into the `admin host` _(from the developer workstation)_ after the `terraform` scripts run to completion. Maybe as part of the setup we can setup `SSH` access from local workstation to the `admin vm` as well and remove `gcloud` dependency. 

Are you suggesting that we create a project also using `terraform`? That still will require the user to go to the console to enable `Cloud Monitoring` for that project. I think requiring the user to have a`Project` created in the console is a reasonable start.

Reply:
I think the gcloud ssh to admin is reasonable for now.